boilerplate/.eslintrc-sample.yml

```diff
@@ -20,6 +20,10 @@ globals:
   Waypoint: true
 
 rules:
+  arrow-body-style:
+    - error
+    - as-needed
+    - requireReturnForObjectLiteral: true
```

Comment:
I noticed that this configuration was missing from our sample ESLint config, so I went ahead and added it. Without it, the Airbnb standard we use enforces this mighty confusing syntax:

```javascript
const thing = () => ({ key: 'value' });
```

That’s supposed to be a function that returns an object literal, which I find to be very confusing because the curly brackets are too easily confused with the brackets used to delimit a function’s block. With the `requireReturnForObjectLiteral` option, this syntax is enforced:

```javascript
const thing = () => {
  return {
    key: 'value',
  };
};
```

That is, in my opinion, much clearer.